.agents/skills/logging-best-practices/scripts/health-check.sh

```
@@ -0,0 +1,18 @@
+#!/bin/bash
+# health-check.sh - Check service health
+# Usage: ./health-check.sh <service_url>
+
+set -euo pipefail
+
+SERVICE_URL="${{1:?Usage: $0 <service_url>}}"
```

Shellcheck (0.11.0)

[error] Parameter expansions can't start with {. Double check syntax.

(SC2296)